dockstore-webservice/src/main/java/io/dockstore/webservice/core/Version.java

```diff
@@ -163,7 +166,18 @@ public void setVerified(boolean verified) {
 
     @ApiModelProperty(value = "Verified source for the version", position = 9)
     public String getVerifiedSource() {
-        return versionMetadata.verifiedSource;
+        Set<String> verifiedSources = new TreeSet<>();
```

Comment:
Hmmm, after looking at this. I am uneasy about the performance implications of these loops being done for every version across every file.  Will need to ponder this longer to see how we can adjust when test parameter files are lost.